src/components/gallery.js

```diff
@@ -15,7 +15,7 @@ type Props = {
   onPicked: (string | number) => *,
   onKeyDown: (SyntheticKeyboardEvent<*>) => *,
   search: string,
-  picked: Array<String | Number>
+  picked: Array<string | number>
```

Comment:
This was annoying, but Flow gave me a super easy to understand message about mixing `String` and `string`, etc. Pretty nice.

Reply:
Later down the line, I ended up making this an `ID` type:
https://github.com/vigetlabs/ars-arsenal/pull/46/files#diff-a2937b260995533a8ad60174b5c3114cR5